tools/number-converter/README.md

```diff
@@ -0,0 +1,30 @@
+# Number converter
+
+This tool converts numbers to binary and hexadecimal as well as dealing with signed and unsigned integers.
```

Comment:
Let's add a link for our future selves to how we do what we do in the readme, similar to: https://github.com/holistic-web/holistic-toolbox/blob/master/tools/json-formatter/README.md#json-formatter